LayoutTests/animations/animation-delay-changed.html

```diff
@@ -47,7 +47,7 @@
 
       const expectedValues = [
         // [animation-name, time, element-id, property, expected-value, tolerance]
-        [null, 1.0, "box1", "left",  200, 10],
+        [null, 0.5, "box1", "left",  100, 10],
```

Comment:
I don't understand the change. Why is it less time-sensitive to check the position halfway through the animation instead of at the end? Also, why don't we need to change the similar test for `box2`.

Reply:
Both `box1` and `box2` animate from 0 to 200.
`box2` test is already testing at 100. It's the middle of the animation.
`box2` animation starts 0.5s after `box1` starts.

The problem of testing at the end of the animation is the left position goes back to 0 when the animation ends.
So, it randomly ends in 0 or 200 at the end of the animation.

Reply:
Revided the commit message.